libpod/container_inspect.go

```diff
@@ -0,0 +1,161 @@
+package libpod
+
+import (
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/projectatomic/libpod/libpod/driver"
+	"github.com/sirupsen/logrus"
+)
+
+func getContainerData(ctr *Container, size bool, driverData *driver.Data) (*ContainerData, error) {
```

Comment:
can you make this func (ctr *Container) getContainerData(...) for consistency with the rest of our container functions?

Reply:
fixed